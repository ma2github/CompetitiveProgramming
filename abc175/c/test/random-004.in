-115914538731754 675838123253311 97520005124515
