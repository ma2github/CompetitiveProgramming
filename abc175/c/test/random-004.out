18394533607239
